GoogleDataTransport/GDTCCTLibrary/GDTCCTUploadOperation.m

```diff
@@ -203,7 +205,7 @@ - (void)uploadTarget:(GDTCORTarget)target withConditions:(GDTCORUploadConditions
             // 429 - Too many requests;
             // 503 - Service unavailable.
             NSInteger statusCode = response.HTTPResponse.statusCode;
-            if (statusCode == 429 || statusCode == 503) {
+            if (statusCode == 429 || statusCode >= 500) {
```

Comment:
All codes >= 500 seems like it could be underspecified?

Reply:
Do you mean we should add the upper boundary for statusCode? Or you would prefer to enumerate specific errors rather than applying the wide range?

Reply:
I don't have a good sense of what codes are feasible and if so what should happen.  Could there be a 511 or a 600 for example. If not, then disregard the comment.

Reply:
The current set of possible errors is pretty limited, but may potentially be extended in the future. Retrying sending the events on any [server error response](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#server_error_responses) looks like a reasonable thing to do and aligns with Android SDK implementation, so I would prefer to keep it as a range.

Thank you for checking.